public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -116,41 +110,338 @@ const Policy: FC<PolicyComponentProps> = ({
   const actualContactPoint = contactPoint ?? inheritedProperties?.receiver ?? '';
   const contactPointErrors = contactPointsState ? getContactPointErrors(actualContactPoint, contactPointsState) : [];
 
+  const childPolicies = currentRoute.routes ?? [];
+  const [showExportDrawer, toggleShowExportDrawer] = useToggle(false);
+  const matchingAlertGroups = matchingInstancesPreview?.groupsMap?.get(currentRoute.id);
+  // sum all alert instances for all groups we're handling
+  const numberOfAlertInstances = matchingAlertGroups
+    ? sumBy(matchingAlertGroups, (group) => group.alerts.length)
+    : undefined;
+
+  // simplified routing permissions
+  const [isSupportedToSeeAutogeneratedChunk, isAllowedToSeeAutogeneratedChunk] = useAlertmanagerAbility(
+    AlertmanagerAction.ViewAutogeneratedPolicyTree
+  );
+  // collapsible policies variables
+  const isThisPolicyCollapsible = useShoudPolicyBeCollapsible(currentRoute);
+  const [isBranchOpen, toggleBranchOpen] = useToggle(false);
```

Comment:
this comment refers to an outdated code. In any case I'm not sure what do you mean? I think all the collapsible nodes should be collapsed by default.

Reply:
This will likely no longer be true when we add the ability to collapse arbitrary policies with child policies, but we can cross that bridge when we get there :)

Reply:
exactly, this is what I thought